src/main/java/core/basesyntax/DataRecord.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class DataRecord {
+    private int supply;
+    private int buy;
+
+    public String getStringSupply() {
+        return String.valueOf(supply);
+    }
+
+    public String getStringBuy() {
+        return String.valueOf(buy);
+    }
```

Comment:
Similar to the 'getStringSupply' method, 'getStringBuy' should be renamed following the Java naming convention, for example, 'getBuyAsString'.